docs/token-contracts/admin-lambda.md

```diff
@@ -0,0 +1,386 @@
+
+Waiting for the node to be bootstrapped before injection...
+Current head: BLJF7Xu3bjHc (timestamp: 2020-09-18T18:38:28-00:00, validation: 2020-09-18T18:38:41-00:00)
+Node is bootstrapped, ready for injecting operations.
+Estimated gas: 114098 units (will add 100 for safety)
+Estimated storage: 163 bytes added (will add 20 for safety)
+Operation successfully injected in the node.
+Operation hash is 'ooFdrXZeF1C4rB56fCkkHdiXXJ1amHX9s8X6NkLTSv52tV99vdK'
+NOT waiting for the operation to be included.
+Use command
+  tezos-client wait for ooFdrXZeF1C4rB56fCkkHdiXXJ1amHX9s8X6NkLTSv52tV99vdK to be included --confirmations 30 --branch BLJF7Xu3bjHchanBUfqoGX96ndm4xUKbpsMkjiPx56ZiM94yeVN
+and/or an external block explorer to make sure that it has been included.
+This sequence of operations was run:
+  Manager signed operations:
+    From: tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm
+    Fee to the baker: ꜩ0.011742
+    Expected counter: 624022
+    Gas limit: 114198
+    Storage limit: 183 bytes
+    Balance updates:
+      tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm ............. -ꜩ0.011742
+      fees(tz1Ke2h7sDdakHJQh8WX4Z372du1KChsksyU,358) ... +ꜩ0.011742
+    Transaction:
+      Amount: ꜩ0
+      From: tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm
+      To: KT1TUb1czajTExzPiSqaPBzDSYf85VqZ5PzT
+      Entrypoint: mint
+      Parameter: (Pair (Pair "tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm" 100) (Pair "TK0" 0))
+      This transaction was successfully applied
+      Updated storage:
+        (Pair (Pair 0x0000aad02222472cdf9892a3011c01caf6407f027081 (Pair 1 18445))
+              (Pair (Pair Unit 18446) (Pair False 18447)))
+      Updated big_maps:
+        Set map(18447)[0] to (Pair 0 (Pair "TK0" (Pair "" (Pair 0 {}))))
+        Set map(18445)[(Pair 0x0000aad02222472cdf9892a3011c01caf6407f027081 0)] to 100
+      Storage size: 4359 bytes
+      Paid storage size diff: 163 bytes
+      Consumed gas: 114098
+      Balance updates:
+        tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm ... -ꜩ0.163
+```
+
+### Use Bob's `AdminLambda` to allow Fred to transfer 5 TK0
+
+Bob transfers `5 TK0` tokens to his `AdminLambda` contract:
+
+```bash
+$ tezos-client transfer 0 from $BOB_ADDRESS to $FA2_ADDRESS \
+                --entrypoint transfer \
+                --arg "{ Pair \"$BOB_ADDRESS\" {Pair \"$ADMIN_LAMBDA\" (Pair 0 5)} }" \
+                --burn-cap 0.067
+
+Waiting for the node to be bootstrapped before injection...
+Current head: BM4xHpzGufJ8 (timestamp: 2020-10-20T19:22:35-00:00, validation: 2020-10-20T19:22:58-00:00)
+Node is bootstrapped, ready for injecting operations.
+Estimated gas: 116167 units (will add 100 for safety)
+Estimated storage: 67 bytes added (will add 20 for safety)
+Operation successfully injected in the node.
+Operation hash is 'oosmJywTSS3JnRbR1HsdL5zu1GQYHby9fCC4q1z3kwW4tvjQNCn'
+Waiting for the operation to be included...
+
+Fatal error:
+  transfer simulation failed
+```
+
+Bob needs to set his `AdminLambda` contract's operator to Fred.
```

Comment:
> Bob needs to set his `AdminLambda` contract's operator to Fred.
Maybe `FA2` contract, not `AdminLambda`?

Reply:
Indeed this is very unclear, and since it is the meat of why we're using an AdminLambda maybe it deserves a few more sentences :)